Makefile

```diff
@@ -56,7 +56,7 @@ ARCHFLAGS := $(ARCHFLAGS)
 
 OPTFLAGS  ?= -g -O
 
-CFLAGS    += -Wall -Wno-unused-function -Wno-write-strings -Wno-sign-compare $(ARCHFLAGS)
+CFLAGS    += -Wall -Wno-unused-function -Wno-write-strings -Wno-sign-compare -std=gnu++11 $(ARCHFLAGS)
```

Comment:
Typically a feature branch (your branch) should not change the compilation process of a project, but I accept the change here.